TS.fsx

```diff
@@ -531,8 +531,25 @@ let EmitDictionaries flavor =
         | "Object" -> Pt.printl "interface %s {" dict.Name
         | _ -> Pt.printl "interface %s extends %s {" dict.Name dict.Extends
 
+        let emitJsonProperty (p: ItemsType.Root) =
+            Pt.printl "%s: %s;" p.Name.Value p.Type.Value
+
+        let removedPropNames = 
+            getRemovedItems ItemKind.Property flavor 
+            |> Array.filter (matchInterface dict.Name)
+            |> Array.map (fun rp -> rp.Name.Value)
```

Comment:
Good point. For some reason I was thinking about `Seq` instead of `Set`, therefore had the impression that `Array.contains` is faster. Will update soon.

I believe `Array.contains` is added in F# 4.0 which was not very well documented.